modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -573,7 +573,7 @@ export const console: ConsoleNS = {
                             }
                         }
                     },
-                    placeholder: "Search by application name"
+                    placeholder: "Search by application name or clientId"
```

Comment:
`client ID`

Reply:
This typo got fixed in a later PR. Please refer to https://github.com/wso2/identity-apps/pull/3498